templates/zerver/api/integration-docs-guide.md

```diff
@@ -71,8 +71,9 @@ Here are a few common macros used to document Zulip's integrations:
 * **Contents:**
   See [source][1].
 
-    **Note:** `{{ integration_display_name }}` is replaced by
-    [Integration.display_name][2] and `{{ recommended_stream_name }}`
+    **Note:** `{` `{ integration_display_name }` `}` is replaced by
+    [Integration.display_name][2] and  
+    `{` `{ recommended_stream_name }` `}`
     is replaced by [Integration.stream_name][3].
```

Comment:
@Privisus: Thanks for working on this! I am not sure if we need this. The text seems to be rendered fine on my end:
![screenshot from 2018-01-12 16-44-23](https://user-images.githubusercontent.com/7251823/34893520-1da8260c-f7b8-11e7-9e3c-cbcdfa366fc4.png)

Could you please check again and see if this is something browser specific maybe? Thanks! :)

Reply:
Maybe rebase and check again! :)